apps/src/applab/ApplabVisualizationColumn.jsx

```diff
@@ -12,32 +23,49 @@ var connect = require('react-redux').connect;
 var ApplabVisualizationColumn = React.createClass({
   propTypes: {
     isReadOnlyWorkspace: React.PropTypes.bool.isRequired,
+    hideSource: React.PropTypes.bool.isRequired,
+    isEmbedView: React.PropTypes.bool.isRequired,
+
     isEditingProject: React.PropTypes.bool.isRequired,
     screenIds: React.PropTypes.arrayOf(React.PropTypes.string).isRequired,
     onScreenCreate: React.PropTypes.func.isRequired,
   },
 
   render: function () {
+    var showFrame = !this.props.isShareView;
+
     return (
-      <div id="visualizationColumn">
+      <div
+          id="visualizationColumn"
+          className={classNames({with_padding: this.props.visualizationHasPadding})}
+          style={[
+            (this.props.isEmbedView || this.props.hideSource) && styles.nonResponsive
+          ]}
+      >
```

Comment:
I notice you've been closing tags on a newline at matching depth, [consistent with airbnb](https://github.com/airbnb/javascript/tree/master/react#tags) but in [contrast to our own styleguide](https://github.com/code-dot-org/code-dot-org/blob/staging/STYLEGUIDE.md#react).  This style has been growing on me too.  Shall we just remove the exception from our styleguide?